caching-service/src/main/java/org/zowe/apiml/caching/api/CachingController.java

```diff
@@ -14,61 +14,66 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
+import org.springframework.http.server.reactive.ServerHttpRequest;
+import org.springframework.http.server.reactive.SslInfo;
 import org.springframework.web.bind.annotation.*;
+import org.zowe.apiml.cache.Storage;
+import org.zowe.apiml.cache.StorageException;
 import org.zowe.apiml.caching.model.KeyValue;
 import org.zowe.apiml.caching.service.Messages;
-import org.zowe.apiml.caching.service.Storage;
-import org.zowe.apiml.caching.service.StorageException;
+import org.zowe.apiml.config.ApplicationInfo;
 import org.zowe.apiml.message.core.Message;
 import org.zowe.apiml.message.core.MessageService;
-
-import jakarta.servlet.http.HttpServletRequest;
+import reactor.core.publisher.Mono;
 
 import java.util.Optional;
 
 @Slf4j
 @RestController
 @RequiredArgsConstructor
-@RequestMapping("/api/v1")
+@RequestMapping("/cachingservice/api/v1")
 public class CachingController {
```

Comment:
given our situation with coverage of negative cases, this controller could use a couple of lines for each endpoint that makes sure there's an authentication in place before continuing (sort of like in ReactivePATController to name one)